src/test/rgw/rgw_multi/tests.py

```diff
@@ -2859,3 +2884,76 @@ def test_sync_single_bucket_to_multiple():
     remove_sync_policy_group(c1, "sync-bucket", bucketA.name)
     remove_sync_policy_group(c1, "sync-group")
     return
+
+def test_topic_notification_sync():
+    zonegroup = realm.master_zonegroup()
+    # enable notification replication.
+    arg = ['--enable-feature=notification_v2']
+    for z in zonegroup.zones:
+        z.modify(z.cluster, arg)
+    zonegroup.modify(zonegroup.master_zone.cluster, arg)
+    zonegroup.period.update(zonegroup.master_zone, commit=True)
+
+    # create topics in each zone.
+    zonegroup_conns = ZonegroupConns(zonegroup)
```

Comment:
sometimes the user info is not syncing before topic creation, which fails topic creation on the non-master zone. 
probably good to add "seep" here

Reply:
i tried that first... for some reason it did not work :-(

Reply:
> i tried that first... for some reason it did not work :-(

i add sleep and zonegroup_checkpoint. and it was working for even without those 2. Lemme know if you want some specific sleep value.

Reply:
will try to run the test again. and let you know

Reply:
working with the "sleep"